methods/rotThirteen.js

```diff
@@ -0,0 +1,24 @@
+
+/**
+ * @file decodeString
+ * @author Alexandre Hamel (a.k.a hamelatoire)
```

Comment:
The original author of the algorithm should be attributed here, not the one creating the file.
https://github.com/TheBastionBot/Bastion/blob/573e0379c64c2846b1c6aec70993f2e9a8b60834/commands/queries/rot13.js#L2-L3